apache/superset
# feat(explore): Replace overlay with alert banner when chart controls change

### SUMMARY
Before, when user made some changes to controls, we displayed an overlay with "Run query" button. This PR removes that overlay and introduces an alert banner that tells the user that some controls have changed and that they should click "Update chart" button. Thanks to that, the user can still view their chart while doing changes to controls.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/15073128/163240324-f09abeb7-588a-4953-b0a4-787bbc83d80d.mov

### TESTING INSTRUCTIONS
1. Create a chart
2. Do some changes in controls and verify that an alert banner showed up.
3. After clicking "Update chart" the banner should disappear.
4. When banner is visible, the chart should still respond to events that trigger rerendering, such as changing the width of browser window.

### ADDITIONAL INFORMATION



CC @kasiazjc
